Source/EventFlow.Tests/IntegrationTests/Sagas/AlternativeSagaStoreTests.cs

```diff
@@ -101,5 +102,15 @@ public void TearDown()
             testAggregate.Bs.Should().Be(1);
             testAggregate.Cs.Should().Be(0);
         }
+
+        [Test]
+        public void SagaLocatorReturningNullDoesntThrow()
+        {
+            // Arrange
+            var aggregateId = AlternativeSagaStoreTestClasses.SagaTestAggregateId.With(Guid.Empty);
+
+            // Act
+            _commandBus.Publish(new AlternativeSagaStoreTestClasses.SagaTestBCommand(aggregateId), CancellationToken.None);
```

Comment:
Is it possible to do an assertion, e.g. is the saga loaded?

Reply:
I updated the InMemorySagaStore to be verifiable.